content/administration/odoo_sh/getting_started/settings.rst

```diff
@@ -48,38 +48,58 @@ In addition, they cannot use the webshell nor have access to the server logs.
 +---------------------+-----------------+-----------+-----------+
-|                     | Shell/SSH       |     X     |     X     |
+|                     | Shell/SSH       |  |green|  |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | Mails           |     X     |     X     |
+|                     | Mails           |  |green|  |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | Settings        |     X     |     X     |
+|                     | Upgrade         |  |green|  |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|Production & Staging | History         |     X     |     X     |
+|                     | Settings        |  |green|  |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | 1-click connect |           |     X     |
+|Production & Staging | History         |  |green|  |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | Logs            |           |     X     |
+|                     | 1-click connect |           |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | Shell/SSH       |           |     X     |
+|                     | Logs            |           |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | Mails           |           |     X     |
+|                     | Shell/SSH       |           |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | Monitoring      |           |     X     |
+|                     | Mails           |           |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | Backups         |           |     X     |
+|                     | Monitoring      |           |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|                     | Settings        |     X     |     X     |
+|                     | Backups         |           |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|Status               |                 |     X     |     X     |
+|                     | Upgrade         |           |  |green|  |
 +---------------------+-----------------+-----------+-----------+
-|Settings             |                 |           |     X     |
+|                     | Settings        |  |green|\*|  |green|  |
 +---------------------+-----------------+-----------+-----------+
+|Status               |                 |  |green|  |  |green|  |
++---------------------+-----------------+-----------+-----------+
+|Settings             |                 |           |  |green|  |
++---------------------+-----------------+-----------+-----------+
```

Comment:
Since you're modifying this table, can you please switch to list tables?
Here it is :)
```
.. list-table::
   :header-rows: 1
   :widths: auto

   * -
     -
     - User
     - Admin
   * - Development
     - History
     - |green|
     - |green|
   * -
     - 1-click connect
     - |green|
     - |green|
   * -
     - Logs
     - |green|
     - |green|
   * -
     - Shell/SSH
     - |green|
     - |green|
   * -
     - Mails
     - |green|
     - |green|
   * -
     - Upgrade
     - |green|
     - |green|
   * -
     - Settings
     - |green|
     - |green|
   * - Production & Staging
     - History
     - |green|
     - |green|
   * -
     - 1-click connect
     -
     - |green|
   * -
     - Logs
     -
     - |green|
   * -
     - Shell/SSH
     -
     - |green|
   * -
     - Mails
     -
     - |green|
   * -
     - Monitoring
     -
     - |green|
   * -
     - Backups
     -
     - |green|
   * -
     - Upgrade
     -
     - |green|
   * -
     - Settings
     - |green|\*
     - |green|
   * - Status
     -
     - |green|
     - |green|
   * - Settings
     -
     -
     - |green|
```